bsyncpy/bsync.py

```diff
@@ -3149,64 +3197,83 @@ class EscalationRate(BSElement):
-        "Association of Energy Engineers Certified Sustainable Development Professional (CSDP)",
-        "Association of Energy Engineers Certified Power Quality Professional (CPQ)",
         "Association of Energy Engineers Certified Demand Side Manager (CDSM)",
+        "Association of Energy Engineers Certified Energy Auditor (CEA)",
+        "Association of Energy Engineers Certified Energy Manager (CEM)",
         "Association of Energy Engineers Certified Energy Procurement Professional (CEP)",
-        "Association of Energy Engineers Certified Lighting Efficiency Professional (CLEP)",
-        "Association of Energy Engineers Certified Measurement & Verification Professional (CMVP)",
         "Association of Energy Engineers Certified GeoExchange Designer Program (CGD)",
-        "Association of Energy Engineers Certified Business Energy Professional (BEP)",
+        "Association of Energy Engineers Certified Green Building Engineer (GBE)",
         "Association of Energy Engineers Certified Industrial Energy Professional (CIEP)",
+        "Association of Energy Engineers Certified Lighting Efficiency Professional (CLEP)",
+        "Association of Energy Engineers Certified Measurement & Verification Professional (CMVP)",
+        "Association of Energy Engineers Certified Power Quality Professional (CPQ)",
+        "Association of Energy Engineers Certified Residential Energy Auditor (REA)",
+        "Association of Energy Engineers Certified Sustainable Development Professional (CSDP)",
         "Association of Energy Engineers Certified Water Efficiency Professional (CWEP)",
-        "Association of Energy Engineers Energy Efficiency Practitioner (EEP)",
-        "Association of Energy Engineers Renewable Energy Professional (REP)",
         "Association of Energy Engineers Distributed Generation Certified Professional (DGCP)",
-        "Association of Energy Engineers Certified Building Energy Simulation Analyst (BESA)",
+        "Association of Energy Engineers Energy Efficiency Practitioner (EEP)",
         "Association of Energy Engineers Performance Contracting and Funding Professional (PCF)",
-        "Association of Energy Engineers Certified Residential Energy Auditor (REA)",
-        "Association of Energy Engineers Certified Building Commissioning Firm Program (CBCF)",
-        "Association of Energy Engineers Certified Green Building Engineer (GBE)",
-        "Association of Energy Engineers Certified Energy Manager (CEM)",
-        "Association of Energy Engineers Certified Energy Auditor (CEA)",
-        "Association of Energy Engineers Certified Building Commissioning Professional (CBCP)",
+        "Association of Energy Engineers Renewable Energy Professional (REP)",
+        "Building Commissioning Association (BCA) Certified Commissioning Professional (CCP)",
+        "Building Commissioning Certification Board (BCCB) Certified Commissioning Professional (CCP)",
```

Comment:
Why are there two CCPs? BCCB and BCA are essentially the same, and I understand from [this bulletin](https://www.bcxa.org/wp-content/uploads/2014/09/CCP-CandidateBulletin_2012v1.pdf) that only BCCB is responsible for the CCP.

Reply:
@ThibaultMarzullo, The 2 enumerations were requested by the city. This could be an oversight on the city's part or "BCA" verses "BCCB" could be meaningful.